_posts/2023-02-22-user-defined-operators.md

```diff
@@ -0,0 +1,598 @@
+---
+layout: post
+published: true
+title: 'Feature Deep-Dive: User-Defined Operators'
+date: '2023-02-22'
+author: Kamil Śliwak, Matheus Aguiar
+category: Explainers
+---
+
+The prominent new feature in [Solidity 0.8.19](https://github.com/ethereum/solidity/releases/tag/v0.8.19)
```

Comment:
```suggestion
The most notable new feature in [Solidity 0.8.19](https://github.com/ethereum/solidity/releases/tag/v0.8.19)
```
might sound better, but just a recommendation

Reply:
I agree that prominent does not do it justice, but @franzihei 's alternative made it too wordy.
I would say this feature is a highlight of the release by calling it "The outstanding feature in..."

Reply:
`most notable` does sound better to me. `outstanding` not so much.

But `highlight` actually sounds even better. How about `The highlight of Solidity 0.8.19 is ...`?